docs/plugin-syntax-dynamic-import.md

```diff
@@ -44,8 +44,12 @@ Currently, `@babel/preset-env` is unaware that using `import()` with [Webpack re
 // webpack config
 const config = {
   entry: [
+    // using core-js@2
     "core-js/modules/es.promise",
```

Comment:
This is already for `core-js@3`: `core-js@2` uses `es6.` instead of `es.`.

Also, those files exist in `core-js@3`: https://unpkg.com/core-js@3.1.2/modules/

Reply:
I was seeing module resolution errors with the (internal) `modules/` based paths after swapping the configuration over to `core-js@3` and found swapping to the `/stable/` variants to be the solution. 

Something else could certainly be the cause of the resolution error if the `es.` version is supported.